Dockerfile

```diff
@@ -1,39 +1,176 @@
+        -DDAEMON_NAME=mariadb \
+        -DFEATURE_SET="community" \
+        -DSYSCONFDIR=/etc \
+        -DSYSCONF2DIR=/etc/my.cnf.d \
+        -DMYSQL_UNIX_ADDR=/run/mysqld/mysqld.sock \
+        -DDEFAULT_CHARSET=utf8mb4 \
+        -DDEFAULT_COLLATION=utf8mb4_general_ci \
+        -DTMPDIR=/tmp \
+        -DCONNECT_WITH_MYSQL=ON \
+        -DCONNECT_WITH_LIBXML2=system \
+        -DCONNECT_WITH_ODBC=OFF \
+        -DCONNECT_WITH_JDBC=OFF \
+        -DCONNECT_WITH_MONGO=OFF \
+        -DCONNECT_WITH_REST=OFF \
+        -DCONNECT_WITH_VCT=OFF \
+        -DCONNECT_WITH_XMAP=OFF \
+        -DCONNECT_WITH_ZIP=OFF \
+        -DPLUGIN_ARCHIVE=NO \
+        -DPLUGIN_AUDIT_NULL=NO \
+        -DPLUGIN_AUTH_0X0100=NO \
+        -DPLUGIN_AUTH_ED25519=NO \
+        -DPLUGIN_AUTH_PAM=NO \
+        -DPLUGIN_AUTH_SOCKET=STATIC \
+        -DPLUGIN_AUTH_TEST_PLUGIN=NO \
+        -DPLUGIN_BLACKHOLE=NO \
+        -DPLUGIN_CONNECT=NO \
+        -DPLUGIN_DAEMON_EXAMPLE=NO \
+        -DPLUGIN_DEBUG_KEY_MANAGEMENT=NO \
+        -DPLUGIN_DIALOG_EXAMPLES=NO \
+        -DPLUGIN_DISKS=NO \
+        -DPLUGIN_EXAMPLE=NO \
+        -DPLUGIN_EXAMPLE_KEY_MANAGEMENT=NO \
+        -DPLUGIN_FEDERATED=NO \
+        -DPLUGIN_FEDERATEDX=NO \
+        -DPLUGIN_FEEDBACK=NO \
+        -DPLUGIN_FILE_KEY_MANAGEMENT=NO \
+        -DPLUGIN_FTEXAMPLE=NO \
+        -DPLUGIN_HANDLERSOCKET=NO \
+        -DPLUGIN_INNOBASE=STATIC \
+        -DPLUGIN_LOCALES=NO \
+        -DPLUGIN_METADATA_LOCK_INFO=NO \
+        -DPLUGIN_MROONGA=NO \
+        -DPLUGIN_PARTITION=STATIC \
+        -DPLUGIN_PERFSCHEMA=STATIC \
+        -DPLUGIN_QA_AUTH_CLIENT=NO \
+        -DPLUGIN_QA_AUTH_INTERFACE=NO \
+        -DPLUGIN_QA_AUTH_SERVER=NO \
+        -DPLUGIN_QUERY_CACHE_INFO=NO \
+        -DPLUGIN_QUERY_RESPONSE_TIME=NO \
+        -DPLUGIN_ROCKSDB=NO \
+        -DPLUGIN_SEQUENCE=STATIC \
+        -DPLUGIN_SERVER_AUDIT=NO \
+        -DPLUGIN_SIMPLE_PASSWORD_CHECK=NO \
+        -DPLUGIN_SPHINX=NO \
+        -DPLUGIN_SPIDER=NO \
+        -DPLUGIN_SQL_ERRLOG=NO \
+        -DPLUGIN_TEST_SQL_DISCOVERY=NO \
+        -DPLUGIN_TEST_VERSIONING=NO \
+        -DPLUGIN_TOKUDB=NO \
+        -DPLUGIN_USER_VARIABLES=STATIC \
+        -DPLUGIN_WSREP_INFO=NO \
+        -DWITH_INNODB_AHI=ON \
+        -DWITH_INNODB_BZIP2=OFF \
+        -DWITH_INNODB_DISALLOW_WRITES=ON \
+        -DWITH_INNODB_EXTRA_DEBUG=OFF \
+        -DWITH_INNODB_LZ4=OFF \
+        -DWITH_INNODB_LZMA=ON \
+        -DWITH_INNODB_LZO=OFF \
+        -DWITH_INNODB_ROOT_GUESS=ON \
+        -DWITH_INNODB_SNAPPY=OFF \
+        -DGRN_EMBED=OFF \
+        -DWITH_MARIABACKUP=OFF \
+        -DWITH_DBUG_TRACE=OFF \
+        -DENABLED_PROFILING=OFF \
+        -DWITH_SYSTEMD=no \
+        -DWITH_WSREP=OFF \
+        -DWITH_UNIT_TESTS=NO \
+        -DDISABLE_SHARED=ON \
+        -DAWS_SDK_EXTERNAL_PROJECT=OFF \
+        -DUPDATE_SUBMODULES=OFF \
+        -DWITH_JEMALLOC=OFF \
+        -DWITH_ZLIB=system && \
+    DESTDIR=../out ninja install
```

Comment:
Ninja still installs the test suite; can shave a few seconds by removing that (If can figure out why).